.github/workflows/codeql.yml

```diff
@@ -0,0 +1,66 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ 'master' ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ 'master' ]
+  schedule:
+    - cron: '4 0 * * 6'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ${{ (matrix.language == 'swift' && 'macos-latest') || 'ubuntu-latest' }}
```

Comment:
Since swift isn't and won't ever be a language in this repository, could we just simplify this section to ubuntu-latest with a 360 minute timeout?

Reply:
https://github.com/google/volley/pull/454/commits/ca4e3ec2d0be10adb6ee85b5d7c99401ee538b51

https://github.com/google/volley/pull/454/commits/c8e410a2982979e38eec651a2cd74eee28cad56e